tutorial03_traceable_layers/app/build.gradle

```diff
@@ -33,7 +33,8 @@ android {
         }
 }
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
     // force debug layer lib is used in packing
-    compile project(path: ':layerlib', configuration: 'debug')
+//    compile project(path: ':layerlib', configuration: 'debug')
```

Comment:
If you leave this in here, perhaps include a comment describing why specifying the variant is no longer required. Feel free to point to this link: https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#variant_dependencies

Reply:
deleted